apps/camera/style/settings.css

```diff
@@ -11,12 +11,15 @@
   background: rgba(0,0,0,0.85);
   overflow: hidden;
   opacity: 1;
-  transition: opacity 150ms linear;
-  will-change: opacity;
+  transition: opacity 150ms linear, visibility 0s;
+  visibility: visible;
+  will-change: opacity, visibility;
```

Comment:
I don't think `visibility` needs to be called out explicitly for the `will-change` property. The only reason this is here is to promote the settings panel to its own layer in the layer tree.